data/World/Forest Temple.json

```diff
@@ -30,6 +33,9 @@
                 has_explosives or Kokiri_Sword or can_use(Dins_Fire)"
         },
         "exits": {
+            "Forest Temple NE Outdoors": "
+                can_use(Iron_Boots) or can_use(Longshot) or 
+                (Progressive_Scale, 2) or (logic_forest_well_swim and can_use(Hookshot))",
```

Comment:
It was brought up that having gold scale as child could skip needing an item for child to kill the bubble in order to reach the other courtyard, but the only thing for child to do over there is the boomerang trick for the gs. With boomerang you can kill babas and get the necessary weapons for the bubble anyway. So all this entrance does is give the playthrough more options as to which items it decides to pick up for getting to that gs as child. It's practically irrelevant as child and actually does nothing at all as adult. I decided to just put the full logic in. The tooltip is not wrong about the well swim trick only possibly being required in MQ. That's still true.

Come to think of it, this change is not even complete b/c the bubble kill is permanent, and in theory you could kill the bubble as adult so that child can get through and use the boomerang on the skulltula as child. So the entrance from NW to High Balconies would have to be wrapped in a here(). Thinking about it some more I'm thinking this change is probably not worth making. Should I just do away with this well exit? All it's for it so the playthrough doesn't pick up unnecessary items.

Reply:
The name on the trick could be misleading, if it alters logic that could affect gameplay. If it would only affect the playthrough picking up "unnecessary" items, I don't think it's worth it.

Reply:
What's not worth it?
1. We can not make any changes, and unnecessary items get picked up in the playthrough.
2. (Requires adding a here() to the high balconies exit, which I missed before.) We can make it so playthrough won't pick up unnecessary items ...
2a) by having the full logic on the well connection.
2b) by only checking for gold scale on the well connection.